src/liquidity_pools/mod.rs

```diff
@@ -0,0 +1,163 @@
+pub mod all_liquidity_pools_request;
+pub mod response;
+pub mod single_liquidity_pool_request;
+
+static LIQUIDITY_POOLS_PATH: &str = "liquidity_pools";
+
+pub mod prelude {
+    pub use super::response::*;
+    pub use super::single_liquidity_pool_request::*;
+}
+
+#[tokio::test]
+
+async fn test_get_all_liquidity_pools() {
```

Comment:
Kunnen we hier meer variaties testen? Je hebt een aantal reserves die je kan meegeven.